src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -497,6 +497,45 @@ def get_custom_registry_credentials(cmd,
     )
 
 
+def build_timers_info(cmd, schedules):
+    TimerTrigger, TriggerStatus = cmd.get_models(
+        'TimerTrigger', 'TriggerStatus')
+    timer_triggers = []
+
+    # Provide a default name for the timer if no name was provided.
+    for index, schedule in enumerate(schedules, start=1):
+        # space?
+        split_schedule = None
+        if ':' in schedule:
+            split_schedule = schedule.split(":")
+        timer_triggers.append(
+            TimerTrigger(
+                name=split_schedule[0] if split_schedule else "defaultTimer" + str(index),
+                status=TriggerStatus.enabled.value,
+                schedule=split_schedule[1] if split_schedule else schedule
+            ))
+    return timer_triggers
+
+
+def remove_timer_trigger(task_name,
+                         timer_name,
+                         timer_triggers):
+    """Remove the timer trigger from the list of existing timer triggers for a task.
+    :param str task_name: The name of the task
+    :param str timer_name: The name of the timer trigger to be removed
+    :param str timer_triggers: The list of existing timer_triggers for a task
+    """
+
+    if not timer_triggers:
+        raise CLIError("No timer triggers exist for the task '{}'.".format(task_name))
+
+    for timer in timer_triggers:
```

Comment:
maybe warning message if the timer didn't exist to begin with so users can self-correct typos or something

Reply:
Good suggestion. See fix